### PR TITLE
Fix/update

### PR DESCRIPTION
**Pull request by yleprincecraft**
## Changelog :newspaper_roll: 
* Link to the dataset URL updated as the website has moved. :1234: 
* Correction of the timezone computation that was incorrect. :calendar: 
* Update Pipfile to use jupyterlab instead of jupyter :sun_with_face: 
* Update Pipfile to use matplotlib instead of plotly :chart_with_upwards_trend: 
* Some code minor refactoring :man_factory_worker:

### CODE REVIEW

File: src/Preprocessing.ipynb [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "\n",
```
[the remark this answers is not in the document]
Comment (PaulineMCraft) [reply src/Preprocessing.ipynb]: Why removing the part 1.3.2 (Interactive visualization)?

[the remark this answers is not in the document]
Comment (PaulineMCraft) [reply src/Preprocessing.ipynb]: (Github refuse to let me put the comment in Main)

File: src/Tools.py [hunk 2]
```diff
@@ -2,60 +2,41 @@
...
-        )
-
-    layout = {'title': 'Zone {:0>3}'.format(zone),
-                      'xaxis': {'title':'Time'},
-                      'yaxis': {'title':'#Clients'},
-                      'font': dict(size=16)}
-
-    iplot({'data': [real, craft, craft_std, dt_sklearn],
-           'layout': layout})
+def plot_zone_results(z, index, reality, craft, craft_std, sklearn=pd.Series()):
```
Comment (PaulineMCraft) [inline src/Tools.py]: It is better to do `sklearn=None`
```
if not sklearn:
    sklearn=pd.Series()
```

Comment (PaulineMCraft) [reply src/Tools.py]: https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

File: src/Preprocessing.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (yleprince) [reply src/Preprocessing.ipynb]: > Why removing the part 1.3.2 (Interactive visualization)?

To be easier to share without running everything :)

[the remark this answers is not in the document]
Comment (yleprincecraft) [reply src/Preprocessing.ipynb]: :heavy_check_mark:

File: src/Tools.py (hunk 2 above)
Comment (yleprincecraft) [reply src/Tools.py]: :heavy_check_mark:

**Pull request merged**